### DISCUSSION TITLE
Malicious code?

### DISCUSSION THREAD

**Discussion by pixelauthor** [inkstitch/inkstitch#1458] [General]
Hi all. Wife asked me to work on converting some logo vectors to DSTs, so naturally I stumbled across this seemingly good (read: free) solution. I'm a designer, so I'm very familiar with Inkscape, vectors, all that jazz. My concern is that when doing my due diligence before installing, this came up: https://www.virustotal.com/gui/file/9e10febb5536b55af13ecdda4324b02c7644d7eb0bd3f7a5820819926857a122?nocache=1

Not sure it this is just lazy copy/paste coding by a dev or if it's intentional, but either way it's a non-starter for me and I wanted to bring it to everyone's attention so you all could check your systems and others like me can be aware that (at least) we have some coders involved in this Git who need to clean up their code before deploying.

**Comment by lexelby** [comment]
Yes, I'm afraid we're quite familiar with this.  They're false positives, of course.  We're working on signed installers that may help convince virus scanners that we're good.

> we have some coders involved in this Git who need to clean up their code before deploying.

I'm sorry, I'm not following.  Could you clarify what you mean by this?
